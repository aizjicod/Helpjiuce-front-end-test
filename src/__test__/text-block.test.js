import { render, screen } from '@testing-library/react';
import TextBlock from '../components/text-block';
import '../setupTests';

it('renders the right h1 component', () => {
  const { container } = render(<TextBlock option={1} text="this is a h1" index={1} />);
  expect(container.querySelector('.heading1')).toBeInTheDocument();
  expect(screen.getByText(/this is a h1/)).toBeTruthy();
  expect(container.firstChild).toMatchSnapshot();
});
it('renders the right p component', () => {
  const { container } = render(<TextBlock option={0} text="this is a p" index={1} />);
  expect(screen.getByText(/this is a p/)).toBeTruthy();
  expect(container.firstChild).toMatchSnapshot();
});
