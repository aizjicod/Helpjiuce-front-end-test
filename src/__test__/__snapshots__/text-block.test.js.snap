// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the right h1 component 1`] = `
<h1
  class="heading1"
>
  this is a h1
</h1>
`;

exports[`renders the right p component 1`] = `
<p>
  this is a p
</p>
`;
